src/operators/repeat.ts

```diff
@@ -76,15 +72,15 @@ class MoreRepeatSubscriber<T> extends Subscriber<T> {
     super(null);
   }
 
-  _next(value: T) {
+  _next(value: T): void {
     this.parent.destination.next(value);
   }
 
-  _error(err: any) {
+  _error(err: any): void {
     this.parent.destination.error(err);
```

Comment:
Sure there are cases. An Observable execution is not always deterministic. Just imagine 
```js
source.map(() => {
  var random = Math.random();
  if (random === SOME_MAGIC_NUMBER) {
    throw new Error();
  } else {
    return random;
  }
})
```

The first execution may complete, but the second may not.